dotCMS/src/main/java/com/dotmarketing/db/listeners/FlushCacheListener.java

```diff
@@ -0,0 +1,28 @@
+package com.dotmarketing.db.listeners;
+
+public abstract class FlushCacheListener implements DotListener {
+    
+    @Override
+    public final boolean equals(Object obj) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Ensure you override both equals() and hashCode()](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5639225)